cont3xt/vueapp/src/utils/Navbar.vue

```diff
@@ -217,6 +214,23 @@ export default {
     },
     reload () {
       window.location.reload();
+    },
+    /* helper functions ---------------------------------------------------- */
+    getHealth () {
+      interval = setInterval(() => {
+        axios.get('api/health').then((response) => {
+          this.healthError = '';
+          if (timeToWait !== minTimeToWait) {
+            timeToWait = minTimeToWait;
+            this.getHealth();
```

Comment:
I think you need a clear above this? Maybe just have a clear at the top of getHealth if interval is set?